### ISSUE TITLE
cannot open shared object file: No such file or directory

### ISSUE THREAD

**Issue by zachdaniel** [bitwalker/picosat_elixir#4]
I've tried to figure this one out for a sold half a day. I've had this issue before and somehow got around it, but I'm at a loss at this point 😢 

Running an Ash application in a dockerfile (kind of a convoluted dockerfile):
```
FROM hexpm/elixir:1.12.2-erlang-24.0.5-ubuntu-xenial-20210114
# install build dependencies
USER root
RUN apt-get update
RUN apt-get install -y git gcc g++ make curl 
RUN apt-get install -y erlang-dev build-essential
RUN curl -sL https://deb.nodesource.com/setup_16.x  | bash -
RUN curl -sL https://dl.yarnpkg.com/debian/pubkey.gpg | apt-key add -
RUN echo "deb https://dl.yarnpkg.com/debian/ stable main" | tee /etc/apt/sources.list.d/yarn.list
RUN apt-get update
RUN apt-get install -y nodejs yarn
ENV MIX_ENV=prod
WORKDIR /platform
COPY ./platform/mix.exs .
COPY ./platform/mix.lock .
COPY ./platform/config config
RUN mix local.hex --force && \
  mix local.rebar --force && \
  mix deps.get && \
  mix deps.compile
WORKDIR /ui
COPY ./ui .
RUN yarn
RUN yarn build
RUN cp -r ../platform/priv/static ./static_assets
WORKDIR /platform
COPY ./platform/lib ./lib
COPY ./platform/rel ./rel
COPY ./platform/priv ./priv
RUN mix compile
RUN cp -r ../ui/static_assets priv/static
RUN mix phx.digest
RUN mix release
CMD ["_build/prod/rel/my_app/bin/my_app", "start"]
```
And I get this when I start the release:

```
=ERROR REPORT==== 29-Aug-2021::06:39:23.460056 ===
Error in process <0.3100.0> on node overcode@5def0b5eafc5 with exit value:
{{badmatch,{error,{load_failed,"Failed to load NIF library: '/platform/_build/prod/rel/overcode/lib/picosat_elixir-0.1.5/priv/picosat_nif.so: cannot open shared object file: No such file or directory'"}}},
 [{picosat_nif,init,0,[{file,"src/picosat_nif.erl"},{line,9}]},
  {init,'-run_on_load_handlers/2-fun-0-',1,[]}]}
```

I've tried ensuring that the path to picosat elixir's priv directory is in `LD_LIBRARY_PATH` but that had no effect.

Any ideas?

**Comment by zachdaniel** [comment]
https://github.com/zachdaniel/docker_issue_example

That example repo shows the issue, w/ `docker build -t example .` and `docker run example`

**Comment by bitwalker** [comment]
This has been fixed in the 0.2.0 release. Thanks for reporting!
